go-controller/pkg/ovn/obj_retry.go

```diff
@@ -1230,5 +1226,9 @@ func (oc *Controller) WatchResource(objectsToRetry *retryObjs) *factory.Handler
 		},
 		syncFunc) // adds all existing objects at startup
 
+	// track the retry entries and every 30 seconds (or upon explicit request) check if any objects
```

Comment:
as we discussed on slack ( just documenting here) handlerAdd will call sync, which may request immediate retry of resources by sending to a channel. The retry routine will not have started yet, but this is OK because the send to the channel is non blocking and when the retry routine finally starts it will retry immediately.